kinto_wizard/async_kinto.py

```diff
@@ -8,34 +8,40 @@
 """
 
 import functools
+import mock
```

Comment:
mock is now part of stdlib ;)

Reply:
Yes I tried to find it didn't know it was encapsulated inside unittest